backend/src/controllers/comparisons/comparisons.controller.ts

```diff
@@ -129,6 +129,38 @@ const getComparisonGeneralInfo = async (
   }
 };
 
+const getActiveComparisonCarsPreview = async (
+  req: TypedRequestBody<{ tokenPayload: TokenPayload }>,
+  res: Response,
+  next: NextFunction,
+): Promise<void> => {
+  try {
+    const comparisonPreview =
+      await comparisonsService.getActiveComparisonCarsPreview(
+        req.body.tokenPayload.sub,
+      );
+    res.status(httpStatus.OK).json(comparisonPreview);
```

Comment:
Why preview? We don't preview anything :eye: We just show the cars in comparison